### PR TITLE
Add 1-liner to ROUGE doc about the range and interpretation

### PR DESCRIPTION
**Pull request by billbai0102**
### Linked issue(s):
Fixes KOL-3805

### What change does this PR introduce and why?
Adds a interpretation 1-liner to the first paragraph

### Please check if the PR fulfills these requirements

- [x] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [x] Relevant tests for the changes have been added
- [X] Relevant docs have been added / updated

### CODE REVIEW

File: docs/metrics/rouge-n.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ metric collection, is a vital metric in the field of natural language processing
 quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts. ROUGE-N
 provides insights into the ability of a system to capture essential content and linguistic nuances, making it an
 important and versatile tool used in many NLP workflows. As the name implies, it is a recall-based metric — a complement
-to the precision-based BLEU score.
+to the precision-based BLEU score. ROUGE-N ranges between 0 and 1, with 0 indicating no overlap and 1 indicating a perfect overlap.
```
Comment (y27choi) [inline docs/metrics/rouge-n.md]: I personally think it fits better after "It assesses the
quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts." this line. What do you think?

Comment (billbai0102) [reply docs/metrics/rouge-n.md]: Yup, that does flow better than what I had previously. I've rearranged the first paragraph to include your comment.

File: docs/metrics/rouge-n.md [hunk 2]
```diff
@@ -7,10 +7,10 @@
 
 ROUGE-N (**R**ecall-**O**riented **U**nderstudy for **G**isting **E**valuation), a metric within the broader ROUGE
 metric collection, is a vital metric in the field of natural language processing and text evaluation. It assesses the
-quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts. ROUGE-N
-provides insights into the ability of a system to capture essential content and linguistic nuances, making it an
-important and versatile tool used in many NLP workflows. As the name implies, it is a recall-based metric — a complement
-to the precision-based BLEU score.
+quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts, and
+ranges between 0 and 1. A score of 0 indicates no overlap between candidate and reference texts, whereas a perfect score of 1 indicates perfect overlap. ROUGE-N provides insights into the ability of a system to capture essential content and
```
Comment (y27choi) [inline docs/metrics/rouge-n.md]: This line seems to exceed the char limit.

**Review by y27choi — APPROVED** [review]
Awesome.  Thanks for making the change.

**Pull request merged**